server/src/index.ts

```diff
@@ -462,7 +460,8 @@ app.get('/runtime-config.js', (_, res) => {
   window['runConfig'] = {
     GITHUB_APP_CLIENT_ID: '${config.githubAppClientId}',
     TELEMETRY_DISABLED: '${config.telemetryDisabled}',
-    GITHUB_APP_NAME: '${config.githubAppName}'
+    GITHUB_APP_NAME: '${config.githubAppName}',
+    GITHUB_APP_WEBHOOKS_SECRET: '${config.githubAppWebhookSecret}'
```

Comment:
with this, you are exposing the secret to all the unauthenticated users, and it's bad idea security-wise 😁